bin/build.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env sh
 
 function build() {
+  cd 'wp-content/themes/eightshift-boilerplate';
```

Comment:
```suggestion
  cd wp-content/themes/eightshift-boilerplate;
```

Should work fine without the quotes iirc

Reply:
👍